release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -733,6 +743,13 @@ The toolbox script is deprecated and support will be removed from a future {prod
 * `platform.vsphere.ingressVIP`
 * `platform.vsphere.network`
 
+[id="ocp-4-13-node-dep-topology-labels"]
+==== Kubernetes topology labels
+
+The commonly used Kubernetes topology labels `failure-domain.beta.kubernetes.io/zone` and `failure-domain.beta.kubernetes.io/region` have been replaced with `topology.kubernetes.io/zone` and `topology.kubernetes.io/region`, respectively. The replacement labels are available starting with Kubernetes 1.17 and {product-title} version 4.4.
```

Comment:
```suggestion
Two commonly used Kubernetes topology labels are being replaced. The `failure-domain.beta.kubernetes.io/zone` label is replaced with `topology.kubernetes.io/zone`. The `failure-domain.beta.kubernetes.io/region` label is replaced with `topology.kubernetes.io/region`. The replacement labels are available starting with Kubernetes 1.17 and {product-title} version 4.4.
```
IBM Style lists "respectively" as a do-not-use term, so I tried to rewrite this part for you to avoid that word. (See the [Word Usage page](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r), and Ctrl+F to "respectively".)